edge-modules/edge-dashboard/Cargo.toml

```diff
@@ -14,7 +16,7 @@ os_info = "1.1.1"
 reqwest = "0.9.18"
 serde = "1.0"
 serde_derive = "1.0"
-serde_json = "1.0"
+serde_json = "1.0.40"
```

Comment:
Why this change?

Reply:
Is there a drawback in having it 1.0.40 instead of 1.0? Just wondering

Reply:
1.0 means any version with major part 1.0 and any build number. let's say you put 1.0.40 and another crate will use 1.0 rust will compile 2 `serde_json` crates. `cargo build` will check the latest version of `serde_json` on crates.io and for the second crate will use the latest one and for your crate still 1.0.40.

But we have Cargo.lock file that specifies all versions of all crates in a workspace. So instead of going on crates.io `cargo build` will use whatever version specified in the file.

So if you specify version without build section, all crates hopefully will use the same version of dependency.

More information in [cargo book](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html)